### PR TITLE
First and untested stab at multi-tenant migration

### PR DESCRIPTION
**Pull request by snoopdave**
This PR adds a new multitenant_migrate.py script that can, if used correctly, migration a Usergrid 2.0 cluster with multiple "tenants" or organizations to Usergrid 2.1 format. See the comments in the header of the file for instructions on how to run.

Associated JIRA issue: https://issues.apache.org/jira/browse/USERGRID-1068

### CODE REVIEW

File: stack/scripts/multitenant_migrate.py [hunk 1]
```diff
@@ -0,0 +1,701 @@
...
+                self.logger.info("Finished...")
+
+                return
+
+
+            if self.init:
+
+                # Init the migration system as this is the first migration done on the cluster
+
+                self.run_database_setup()
```
Comment (michaelarusso) [inline stack/scripts/multitenant_migrate.py]: Should we also run self.run_database_bootstrap() here?

Comment (snoopdave) [inline stack/scripts/multitenant_migrate.py]: Good catch. I just added it in and tested it. I wonder how/why things worked before, I thought bootstrap was a requirement.

Comment (michaelarusso) [inline stack/scripts/multitenant_migrate.py]: I think if  you've already ran it manually at least once, then you're fine.  It sets up the default usergrid org, superuser acct, and the test org/account

**Comment by michaelarusso** [comment]
:+1:

**Pull request merged**